### PR TITLE
Configurable profile dependencies

### PR DESCRIPTION
**Pull request by benjy**

### CODE REVIEW

**Comment by benjy** [comment]
test this please

**Comment by benjy** [comment]
test this please

File: agov/agov.info.yml [hunk 1]
```diff
@@ -42,6 +42,16 @@ dependencies:
   # Custom modules
   - agov_standard_page
 
+configurable_dependencies:
```
Comment (Sam152) [inline agov/agov.info.yml]: Is this left over?

File: agov/agov_test_profile/agov_test_profile.install [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?php
```
Comment (Sam152) [inline agov/agov_test_profile/agov_test_profile.install]: Is there any chance this profile could live in the project root? Seems like something which would be useful to not package with the profile? At that stage the tests would probably also be coupled with the project root and not packaged for release.

**Comment by benjy** [comment]
Sam, I think this is ready.

**Pull request merged**